### DISCUSSION TITLE
Option to change album cover photo

### DISCUSSION THREAD

**Discussion by doceruna** [FossifyOrg/General-Discussion#407] [Ideas]
I would like to request the ability to change the cover photo of albums in the Gallery app with a custom photo, similar to the Samsung Gallery app. This would allow users to personalize their albums and make them more visually appealing.

Currently, the Gallery app only allows users to use the first photo in an album as the cover photo. This is not always ideal, as the first photo may not be the best representation of the album's contents.

Allowing users to choose a custom cover photo would give them more control over the appearance of their albums. It would also make it easier to identify albums at a glance.

I am a big fan of the Gallery app and it has made the transition from OneUI to LineageOS much easier. <3

I hope you will consider adding this feature in a future update. ^^

**Comment by Aga-C** [comment]
You can do this by long pressing the folder -> three dots menu -> `Change cover image` -> `Select photo`.
